examples/nodejs/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "appengine-alloydb",
+  "description": "Node.js sample for AlloyDB on App Engine.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/alloydb-auth-proxy.git"
+  },
+  "engines": {
+    "node": ">=12.0.0"
```

Comment:
I'm getting a lot of mixed messages in what is the node version supposed to be supported by this sample, here I see it needs to be at least greater than 12 but in the [`sample-test` action](https://github.com/GoogleCloudPlatform/alloydb-auth-proxy/pull/205/files#diff-438f118e98fa390613566cfaaf379918e9590dd007c9934045ea2ab807b4f0b5R140) I see it targetting node 14 and in the [`app.standard.yaml`](https://github.com/GoogleCloudPlatform/alloydb-auth-proxy/pull/205/files#diff-2461ae8d28014de91dd6442d5618338d5be82e1a5c0d6dd61342699c324d2f93R15) it appears to be targetting 16?

Maybe it's going to be simpler to introduce this sample with a higher minimum version requirement given that both version 14 and 16 are already in maintenance-only mode and version 18 is the current LTS, ref: https://github.com/nodejs/Release#release-schedule